specification/common-types/resource-management/v3/privatelinks.json

```diff
@@ -78,9 +80,30 @@
       ],
       "x-ms-enum": {
         "name": "PrivateEndpointConnectionProvisioningState",
-        "modelAsString": true
-      },
-      "readOnly": true
+        "modelAsString": true,
```

Comment:
It's not readonly anymore? It's also a breaking change.

Reply:
is it? can you confirm this?
in this particular case the readonly is not on the enum (that is this definition) but the readonly is present when the enum is used, I don't think we can put the readonly on the enum definition

Reply:
Oh yes you are right.

Reply:
I need to confirm this so unresolve it for now.

Reply:
did you have any confirmation on this?